README.md

```diff
@@ -272,6 +309,20 @@ For running e2e test locally you should:
 4) Run tests with `yarn test:e2e-local`
 
 
+### Storybook testing
```

Comment:
y not make this part of the presentational layer testing above?

also when i run `yarn storybook:test` i get `zsh:1: permission denied: src/components/Button/Button.stories.test.ts` after the tests run successfully.